tool/e2e-tests/helpers.ts

```diff
@@ -28,7 +28,9 @@ import {
   backButton,
   configButton,
   configLink,
-  configPageTitle
+  configPageTitle,
+  claimPageTitle,
+  claimButton
 } from "./locators"
```

Comment:
think it might be good to look at splitting up these locators based on page to avoid having this mega file

Reply:
agreed, couldnt even tell you what they do. maybe in another pr?